test/socket.js

```diff
@@ -479,6 +479,7 @@ describe('Socket', () => {
 
             await server.start();
             const client = new Ws('http://localhost:' + server.info.port);
+            client.onerror = Hoek.ignore;
```

Comment:
Are these still necessary?

Reply:
Yes, if there's no error handler attached it will throw a `read ECONNRESET` error.